app/src/main/res/values-it/strings.xml

```diff
@@ -0,0 +1,292 @@
+    <string name="text_slide2">Puoi scansionare un codice QR usando QR Scanner o generare un codice QR con QR generator.</string>
+    <string name="slide3_heading">Più caratteristiche</string>
+    <string name="text_slide3">Il codice QR che hai scansionato può essere salvato nella Cronologia. Inoltre puoi condivederlo o copiarlo.</string>
+    <string name="wecome">Benvenuto</string>
+    <string name="skip">Salta</string>
+    <string name="next">Continua</string>
+    <string name="help_">Aiuto</string>
+
+    <string name="tutorial">Tutorial</string>
+
+    <!--###ABOUT###-->
+    <string name="about_author">Autori: </string>
+    <string name="about_author_names" translatable="false">Christopher Beckmann, Bassel Hanna, Philipp Rack</string>
+    <string name="about_author_string">%s e collaboratori.</string>
+    <string name="about_affiliation">In affiliazione con</string>
+    <string name="privacy_friendly">Questa applicazione appartiene al gruppo delle applicazioni PRIVACY FRIENDLY sviluppate dalla Technische Universität Darmstadt. Il codice sorgente è concesso in licenza GPLv3. Il copyright per le immagini appartiene alla TU di Darmstadt e Google Inc.</string>
+    <string name="more_info">Maggiori informazioni sono disponibili qui:</string>
+    <string name="github" translatable="false"><a href="https://github.com/SecUSo/privacy-friendly-qr-scanner">Github-Repo</a></string>
+    <string name="url_secuso" translatable="false"><a href="https://www.secuso.org/pfa">SECUSO-Website</a></string>
+    <string name="about_libraries">Librerie utilizzate:</string>
+    <string name="lib_url" translatable="false"><a href="https://github.com/zxing">ZXing</a></string>
+    <string name="app_name_long">Privacy Friendly QR Scanner</string>
+    <string name="version_number">Versione v%s</string>
+    <string name="delete_items">Cancella elementi</string>
+
+    <string name="scan_text">Posiziona questa finestra sopra il codice che vuoi scansionare.</string>
+    <string name="action_clear">Cancella Cronologia</string>
+    <string name="cannot_clear">La cronologia è già vuota</string>
+    <string name="knowRisks">Ho controllato il collegamento e voglio visitare il sito.</string>
+    <string name="domain">Dominio</string>
+    <string name="urlExplanation">I codici QR possono contenere collegamenti, che conducono a falsi siti web, capaci di compromettere i tuoi dati personali (siti di phishing) o trasferire malware (ad esempio virus) al tuo dispositivo.\n\nCome conseguenza dovresti <b>controllare</b> se il seguente collegamento ti sembra corretto:</string>
+    <string name="further_info"><a href="https://www.secuso.informatik.tu-darmstadt.de/de/research/results/nophish-flyer-deutsch/unterstuetzung-bei-der-erkennung-von-phishing-angriffen/">Informazione</a> come controllare i collegamenti forniti dai colleghi della TU Darmstadt. Se non ti fidi del collegamento, clicca non mi fido &amp; non salvare.</string>
```

Comment:
Same a before: not "collegamento", but link

Reply:
done